src/ResourceManager/Sql/Commands.Sql/help/Get-AzureRmSqlDatabaseLongTermRetentionBackup.md

```diff
@@ -0,0 +1,332 @@
+ServerName           : server01
+ServerCreateTime     : 2/29/2018 12:12:19 AM
+```
+
+This command gets the backup with name 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+
+### Example 3: Get all long term retention backups for a database
+```powershell
+PS C:\> Get-AzureRmSqlDatabase -ResourceGroupName resourcegroup01 -ServerName server01 -DatabaseName database01 | Get-AzureRmSqlDatabaseLongTermRetentionBackup
+
+
+BackupExpirationTime : 3/22/2018 5:50:55 AM
+BackupName           : 601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+BackupTime           : 3/15/2018 5:50:55 AM
+DatabaseName         : database01
+DatabaseDeletionTime :
+LocationName         : northeurope
+ResourceId           : /subscriptions/371edd6d-9630-4558-a7bd-ee139498e6a1/providers/Microsoft.Sql/locations/northeurope/longTermRetentionServers/server01/longTermRetentionDatabases/database01/longTermRetentionBackups/601061b7-d10b-46e0-bf77-a2bfb16a6add;131655666550000000
+ServerName           : server01
+ServerCreateTime     : 2/29/2018 12:12:19 AM
+```
+
+This command gets all long term retention backups for database01
+
+## PARAMETERS
+
+### -BackupName
+The name of the backup.
+
+```yaml
+Type: String
+Parameter Sets: BackupName, GetBackupByResourceId, GetBackupByInputObject
+Aliases:
+
+Required: True
+Position: 3
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DatabaseName
+The name of the Azure SQL Database the backup is from.
+
+```yaml
+Type: String
+Parameter Sets: DatabaseName, BackupName
+Aliases:
+
+Required: True
+Position: 2
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DatabaseState
+The state of the database whose backups you want to find, Alive, Deleted, or All.
+Defaults to All
+
+```yaml
+Type: String
+Parameter Sets: LocationName, ServerName, DatabaseName, GetBackupsByResourceId, GetBackupsByInputObject
+Aliases:
+Accepted values: All, Deleted, Live
+
+Required: False
+Position: 4
+Default value: None
+Accept pipeline input: True (ByPropertyName)
+Accept wildcard characters: False
+```
+
+### -DefaultProfile
+The credentials, account, tenant, and subscription used for communication with Azure.
+
+```yaml
+Type: IAzureContextContainer
+Parameter Sets: (All)
+Aliases: AzureRmContext, AzureCredential
+
+Required: False
+Position: Named
+Default value: None
+Accept pipeline input: False
+Accept wildcard characters: False
+```
+
+### -InputObject
+The Database object to get backups for.
+
+```yaml
+Type: AzureSqlDatabaseModel
+Parameter Sets: GetBackupByInputObject, GetBackupsByInputObject
+Aliases:
+
+Required: True
+Position: 0
+Default value: None
+Accept pipeline input: True (ByValue)
+Accept wildcard characters: False
+```
+
+### -LocationName
+The location the backups are in.
```

Comment:
Can we also get a pretty comment here about the databases (alive or deleted) locations?

Reply:
Fixed in next commit.